plugins/github-actions/src/api/GithubActionsClient.ts

```diff
@@ -102,4 +102,22 @@ export class GithubActionsClient implements GithubActionsApi {
     });
     return run.data;
   }
-}
+  async downloadJobLogsForWorkflowRun({
+    token,
+    owner,
+    repo,
+    runId,
+  }: {
+    token: string;
+    owner: string;
+    repo: string;
+    runId: number;
+  }): Promise<any> {
```

Comment:
Could you avoid `any` here?

Reply:
This is a good call out. I plugged in `Promise<EndpointInterface>` and have good results. Will place in the update.